examples/adminapi.py

```diff
@@ -237,7 +242,12 @@ def example_delete_acls(a, args):
         )
     ]
 
-    fs = a.delete_acls(acl_binding_filters, request_timeout=10)
+    fs = None
```

Comment:
There is no need for this since there is no case where fs has not been assigned when it is used later.

Same comment for the other locations